apps/fz_http/priv/repo/migrations/20221207191602_add_defaults_to_configurations.exs

```diff
@@ -0,0 +1,20 @@
+defmodule FzHttp.Repo.Migrations.AddDefaultsToConfigurations do
+  use Ecto.Migration
+
+  def change do
+    alter table(:configurations) do
+      modify(:id, :uuid, default: fragment("gen_random_uuid()"))
+      add(:default_client_mtu, :integer, null: false, default: 1280)
+      add(:default_client_persistent_keepalive, :integer, null: false, default: 0)
+      add(:default_client_allowed_ips, :string, null: false, default: "0.0.0.0/0, ::/0")
+      add(:default_client_dns, :string, null: false, default: "1.1.1.1, 1.0.0.1")
```

Comment:
What if a user wants to omit the `DNS`, `MTU`, `PersistentKeepalive`, or `AllowedIPs` fields from the generated configs entirely?

Reply:
Made these columns nullable.